grpcpp_channelz.vcxproj.filters

```diff
@@ -0,0 +1,42 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
this one added by mistake?

Reply:
dammit, that keeps happening; git hates me